pyproject.toml

```diff
@@ -111,7 +111,7 @@ __version_tuple__ = version_tuple = {version_tuple!r}
 
 [tool.blue]
 line_length = 99
-target-version = ["py37"]
+target-version = ["py38"]
```

Comment:
Doesn't blue understand `requires-python = ">=3.8"`?

Reply:
It does not. We'll probably switch to ruff, as blue's development seems done.

Reply:
Then I will provide a PR to switch to ruff with [`quote-style`](https://docs.astral.sh/ruff/settings/#format-quote-style) set to `single`.

Reply:
That would be great!